### DISCUSSION TITLE
Why use yandex/clickhouse-server when there is an official clickhouse/clickhouse-server image?

### DISCUSSION THREAD

**Discussion by FarisZR** [plausible/analytics#1818] [Self-Hosted Support] [answered]
I noticed that the docker-compose.yml file in the hosting repo uses a Yandex image for clickhouse
but clickhouse has an [official docker image](https://hub.docker.com/r/clickhouse/clickhouse-server), and i swtichted to it without any problems.

Is there a specific reason for choosing the Yandex image?

**Comment by ukutaht** [comment, marked as answer]
When the docker-compose file was written Clickhouse Inc didn't exist yet. We should switch to the new official image in the next release.
